playbooks/elasticsearch.yml

```diff
@@ -0,0 +1,61 @@
+---
+# Copyright 2015, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Setup ElasticSearch host
+  hosts: elasticsearch_all
+  user: root
+  pre_tasks:
+    - name: Create ElasticSearch data directory on host
+      file:
+        path: "/openstack/{{ container_name }}/var/lib/elasticsearch"
+        state: directory
+        group: "root"
+        owner: "root"
+        mode:  "0755"
+        recurse: no
+      delegate_to: "{{ physical_host }}"
+      when: is_metal == false or is_metal == "False"
+    - name: ElasticSearch extra lxc config
+      lxc_container:
```

Comment:
should be lxc-container: rather than _, otherwise:
# openstack-ansible elasticsearch.yml
Variable files: "-e @/etc/openstack_deploy/user_extras_secrets.yml -e @/etc/openstack_deploy/user_extras_variables.yml -e @/etc/openstack_deploy/user_variables.yml "
ERROR: lxc_container is not a legal parameter in an Ansible task or handler